src/lib/conversions.ts

```diff
@@ -4,7 +4,7 @@ import { EnvVarObject } from './types';
 
 export function envArrayToObject(env: string[]): EnvVarObject {
 	const toPair = (keyVal: string) => {
-		const m = keyVal.match(/^([^=]+)=\s*(.*)\s*$/);
+		const m = keyVal.match(/^([^=]+)=((?:.|\n|\r)*)$/);
```

Comment:
This still doesn't handle all the newline-ish characters, check https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#Special_characters_meaning_in_regular_expressions which recommends
```suggestion
		const m = keyVal.match(/^([^=]+)=([^]*)$/);
```
if you don't need old IE (I think my previous suggestion of `[\s\S]` works in old IE as it's what is usually suggested but is maybe less performant and that's why mdn has the other suggestion?)